pkg/cli/webhook/gitlab.go

```diff
@@ -42,7 +42,7 @@ func (gl *gitLabConfig) Run(_ context.Context, opts *Options) (*response, error)
 }
 
 func (gl *gitLabConfig) askGLWebhookConfig(controllerURL, apiURL string) error {
-	msg := "Please enter the project ID for the repository you want to be configured :"
+	msg := "Please enter the project ID for the repository you want to be configured \n  Hint: project ID refers to an unique ID shown at the top of your GitLab project. eg: \"524897\":"
```

Comment:
do we need to write it as `Hint:` ? 😅 
We can just directly say `You can find the project id ...`
i don't have strong opinion to remove it tho

Reply:
To give information on where to find projectID so i called it as Hint
But if you feel `Hint` could be given in tricky situation we can remove `Hint` word

Let me know

Reply:
Updated changes as well as command output in description  
PTAL